pyanaconda/modules/payload/dnf/packages/packages.py

```diff
@@ -0,0 +1,377 @@
+        """Set list of excluded packages from the installation.
+
+        :param exclude_package_list: list of excluded packages
+        :type exclude_package_list: [str]
+        """
+        self._exclude_package_list = exclude_package_list
+        self.exclude_package_list_changed.emit(self._exclude_package_list)
+        log.debug("The exclude_package_list has changed")
+
+    @property
+    def exclude_docs(self):
+        """Should the documentation be excluded during the installation.
+
+        :rtype: bool
+        """
+        return self._exclude_docs
+
+    def set_exclude_docs(self, exclude_docs):
+        """Set if the documentation should be installed with the packages.
+
+        :param exclude_docs: True if packages documentation should be installed
+        :type exclude_docs: bool
+        """
+        self._exclude_docs = exclude_docs
+        self.exclude_docs_changed.emit(self._exclude_docs)
+        log.debug("The exclude_docs has changed")
+
+    @property
+    def exclude_weakdeps(self):
+        """Should the packages weak dependencies be excluded from the installation.
+
+        :rtype: bool
+        """
+        return self._exclude_weakdeps
+
+    def set_exclude_weakdeps(self, exclude_weakdeps):
+        """Set if the weak dependencies should be excluded during the installation.
+
+        :param exclude_weakdeps: True if the weak dependencies should be excluded
+        :type exclude_weakdeps: bool
+        """
+        self._exclude_weakdeps = exclude_weakdeps
+        self.exclude_weakdeps_changed.emit(self._exclude_weakdeps)
+        log.debug("The exclude_weakdeps has changed")
+
+    @property
+    def ignore_missing(self):
+        """Ignore packages that are missing from the repositories.
+
+        :rtype: bool
+        """
+        return self._ignore_missing
+
+    def set_ignore_missing(self, ignore_missing):
+        """Set if the packages missing during the installation should be ignored.
+
+        :param ignore_missing: True if missing packages should be ignored.
+        :type ignore_missing: bool
+        """
+        self._ignore_missing = ignore_missing
+        self.ignore_missing_changed.emit(self._ignore_missing)
+        log.debug("The ignore_missing has changed")
+
+    @property
+    def install_languages(self):
+        """List of language packs marked for installation.
+
+        :rtype: [str]
+        """
+        return self._install_languages
+
+    def set_install_languages(self, install_languages):
+        """Set list of language packs marked for installation.
+
+        :param install_languages: list of languages
+        :type install_languages: [str]
+        """
+        self._install_languages = install_languages
+        self.install_languages_changed.emit(self._install_languages)
+        log.debug("The install_languages has changed")
+
+    @property
+    def multi_lib(self):
+        """Enable 'all' multilib policy as opposed to the default of “best”.
```

Comment:
Based on the docstring, it might be called `multilib_policy` and return strings "best" or "all". Otherwise, the space between multi and lib looks weird. [DNF](https://dnf.readthedocs.io/en/latest/api_conf.html#dnf.conf.Conf.multilib_policy) uses `multilib`.

Reply:
Fixed.